rs/nns/governance/src/neuron_store/metrics.rs

```diff
@@ -129,7 +129,9 @@ impl NeuronSubsetMetrics {
             neuron.staked_maturity_e8s_equivalent.unwrap_or_default();
         let maturity_e8s_equivalent = neuron.maturity_e8s_equivalent;
 
-        let voting_power = neuron.voting_power(now_seconds);
+        // TODO: Also provide deciding voting power. Ideally, we'd rename the metrics to
+        // potential_voting_power, but that's probably not worth it.
+        let voting_power = neuron.potential_voting_power(now_seconds);
```

Comment:
Ideally, we'd probably want to have both metrics (potential_voting_power, deciding_voting_power) to evaluate the effect of this feature.

NCR

Reply:
Yeah, sure. Already had a ticket for metrics. Should have mentioned it here.